web/experimental/embed-new.html

```diff
@@ -72,8 +72,13 @@
     </div>
     <div id="console-view" class="tabview flex-auto p-2" hidden></div>
 </section>
+<footer flex layout horizontal class="footer">
+    <div id="issues-message"></div>
+    <span class="issues-spacer"></span>
```

Comment:
Is `issues-spacer` still being used? It's not in the test version of the HTML, and I don't see it in the CSS changes here.